bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -142,6 +142,11 @@
 
           let(:dependency_files) { bundler_project_dependency_files("bundler_specified") }
 
+          before do
+            stub_request(:get, rubygems_url + "versions/bundler.json").
+              to_return(status: 200, body: fixture("rubygems_responses", "versions-bundler.json"))
```

Comment:
Does this mean we make an extra API call for every update now?

Reply:
Nah, this just seemed to pop up in this test which specifically updates `bundler`, but going to revert to see if this still fails with the `bundle exec` fix